code/game/objects/items/weapons/implants/implant_explosive.dm

```diff
@@ -133,6 +133,8 @@
 /obj/item/weapon/implant/dust
 	name = "duster implant"
 	desc = "An alarm which monitors host vital signs, transmitting a radio message and dusting the corpse on death."
+	icon = 'icons/effects/blood.dmi'
+	icon_state = "remains"
 
```

Comment:
the PR changes the action button system to allow for custom action button icons
that is exactly what this needs